path_linux.go

```diff
@@ -53,6 +53,14 @@ func pathSysBusPciDevices() string {
 	return filepath.Join(pathRoot(), "sys", "bus", "pci", "devices")
 }
 
+func pathSysClassDrm() string {
+	return filepath.Join(pathRoot(), "sys", "class", "drm")
+}
+
+func pathSysClassNet() string {
+	return filepath.Join(pathRoot(), "sys", "class", "net")
+}
```

Comment:
Cool, thanks for moving these. I'd obviously missed these when I did the new path_linux.go module. ++